src/olympia/amo/tests/test_settings.py

```diff
@@ -48,15 +48,15 @@ def test_raven_release_config():
         os.remove(version_json)
 
     # by default, if no version.json exists it simply fetches a git-sha
-    assert len(get_raven_release()) == 40
+    assert len(get_sentry_release()) == 40
 
     # It fetches `version` from the version.json
     with open(version_json, 'w') as fobj:
         fobj.write(json.dumps({
             'version': '2018.07.19'
```

Comment:
Nit: Can we make this a const so that it doesn't require a double hardcoding? The same goes for `commit = '1111111'`, below.